.github/workflows/e2e-test.yml

```diff
@@ -56,7 +56,6 @@ jobs:
       - name: Setup test-website project for Yarn v2
         run: |
           cd test-website
-          yarn set version berry
```

Comment:
Was wondering why it's removed? 
Are we sure to use Yarn2 in this flow without that line?

Reply:
ah ok it's been moved :)

Reply:
It has been moved a few step earlier, as stated in the summary to ensure that `install` is also performed by v2.